src/components/pages/landing/Landing.js

```diff
@@ -1,44 +1,59 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { Link } from 'react-router-dom';
+import SocialFooter from './SocialFooter';
 
 class Landing extends React.Component {
+  componentDidMount(){
+    this.script = document.createElement('script');
+    this.script.type = 'text/javascript';
+    this.script.async = true;
+    this.script.innerHTML = "window.liveSettings = {picker: '#transifexTranslateElement',api_key: '9eda410a7db74687ba40771c56abd357',detectlang: false,site: 'gfw-watcher'};";
```

Comment:
Please move the api_key and use a env var instead, also I would save each script and config options in a constant, to separate it from the component.